pkg/ddevapp/router.go

```diff
@@ -137,6 +149,15 @@ func generateRouterCompose() (string, error) {
 
 	uid, gid, username := util.GetContainerUIDGid()
 
+	var ephemeralPorts []string
+	if ephemeralRouterHTTPPort != "" && originalRouterHTTPPort != "" {
+		ephemeralPorts = append(ephemeralPorts, strings.Join([]string{originalRouterHTTPPort, ephemeralRouterHTTPPort}, ":"))
+	}
+	if ephemeralRouterHTTPSPort != "" && originalRouterHTTPSPort != "" {
+		ephemeralPorts = append(ephemeralPorts, strings.Join([]string{originalRouterHTTPSPort, ephemeralRouterHTTPSPort}, ":"))
+	}
+	ephemeralPortsStr := strings.Join(ephemeralPorts, ",")
```

Comment:
This strategy can't work at all, because one project may have a different router_http*_port from another, and this technique overwrites the environment variable in the router, losing track of the first project's information I'm pretty sure this strategy can't work in that case at all.